src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -34,7 +34,7 @@ import {
 } from './MyVideos.styles'
 import { NewVideoTile } from './NewVideoTile'
 
-const TABS = ['All Videos', 'Public', 'Drafts', 'Unlisted'] as const
+const TABS = ['All videos', 'Public', 'Drafts', 'Unlisted'] as const
```

Comment:
wow at the amount of changes this required, we need to refactor Tabs component 😱

Reply:
I stored `currentTabName === 'All Videos'` in a variable to not repeat myself. For now, I think it should be enough.